examples/operator-rbac/1-setup-operator-rbac.ipynb

```diff
@@ -0,0 +1,753 @@
+      "      server: {}\r\n",
+      "    registry:\r\n",
+      "      local:\r\n",
+      "        server: {}\r\n",
+      "    ui: {}\r\n",
+      "featurestore.feast.dev/sample-kubernetes-auth created\r\n"
+     ]
+    }
+   ],
+   "execution_count": 156
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Validate the running FeatureStore deployment\n",
+    "Validate the deployment status."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-05T18:49:07.735147Z",
+     "start_time": "2025-03-05T18:48:47.817817Z"
+    }
+   },
+   "source": [
+    "!kubectl get all\n",
+    "!kubectl wait --for=condition=available --timeout=8m deployment/feast-sample-kubernetes-auth"
+   ],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME                                                READY   STATUS     RESTARTS   AGE\r\n",
+      "pod/feast-sample-kubernetes-auth-774f6df8df-9ngcl   0/4     Init:0/1   0          5s\r\n",
+      "\r\n",
+      "NAME                                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE\r\n",
+      "service/feast-sample-kubernetes-auth-offline    ClusterIP   10.96.2.92      <none>        80/TCP    5s\r\n",
+      "service/feast-sample-kubernetes-auth-online     ClusterIP   10.96.128.237   <none>        80/TCP    5s\r\n",
+      "service/feast-sample-kubernetes-auth-registry   ClusterIP   10.96.209.35    <none>        80/TCP    5s\r\n",
+      "service/feast-sample-kubernetes-auth-ui         ClusterIP   10.96.76.211    <none>        80/TCP    5s\r\n",
+      "\r\n",
+      "NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE\r\n",
+      "deployment.apps/feast-sample-kubernetes-auth   0/1     1            0           5s\r\n",
+      "\r\n",
+      "NAME                                                      DESIRED   CURRENT   READY   AGE\r\n",
+      "replicaset.apps/feast-sample-kubernetes-auth-774f6df8df   1         1         0       5s\r\n",
+      "deployment.apps/feast-sample-kubernetes-auth condition met\r\n"
+     ]
+    }
+   ],
+   "execution_count": 157
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Validate that the FeatureStore CR is in a `Ready` state."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-05T18:49:15.695148Z",
+     "start_time": "2025-03-05T18:49:15.499064Z"
+    }
+   },
+   "source": [
+    "!kubectl get feast"
+   ],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME                     STATUS   AGE\r\n",
+      "sample-kubernetes-auth   Ready    34s\r\n"
+     ]
+    }
+   ],
+   "execution_count": 158
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "## Configure the RBAC Permissions\n",
```

Comment:
This section code be a bit more descriptive on the intentions, to clarify what the code is doing

Reply:
added more details and comments in code as well to make it clear.